products/mattermost.md

```diff
@@ -13,7 +13,6 @@ releaseDateColumn: true
 
 auto:
   methods:
-  -   git: https://github.com/mattermost/mattermost-server.git
   -   release_table: https://docs.mattermost.com/about/mattermost-server-releases.html
```

Comment:
```suggestion
  -   github_releases: mattermost/mattermost-server
  -   release_table: https://docs.mattermost.com/about/mattermost-server-releases.html
```

We should switch to `github_releases` instead: prerelease are automatically excluded with this auto method.